subprojects/docs/src/docs/userguide/buildCache.adoc

```diff
@@ -342,6 +342,15 @@ the credentials the api:org.gradle.caching.http.HttpBuildCache[] uses to access
 </sample>
 ++++
 
+It is also possible to configure the build cache from an <<sec:using_an_init_script,init script>>, which can be added to your Gradle user home or your custom Gradle distribution.
```

Comment:
I would mention using it from the command-line, too.

Reply:
I added the command line use case.